docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/analytics/analytics.js

```diff
@@ -163,14 +163,42 @@ $(document).ready(function () {
         $(el).on('click.analytics', 'a', function (event) {
             var $this = $(this);
             var $borderlessContainer = $(this).parents('.borderless-container');
-            var cardTitle = $borderlessContainer.find('h2').first().text();
+            var cardTitle = $borderlessContainer.find('h2').first().text().trim();
             var linkText = cardTitle;
             var container = 'fullwidth';
             var containerIndex = i + 1;
             NCIAnalytics.CardClick(this, cardTitle, linkText, container, containerIndex);
         });
     });
 
```

Comment:
Can we attempt to do this in plain javascript since it is new?

Reply:
If it's not a showstopper I would lean toward sticking with the existing precedent for this one.  Moving forward we can emphasize using plain javascript where possible for upcoming analytics changes, if that is the direction we want to go

Reply:
agreed.  Bryan said he would be okay with that, too.  Will address at a later time, like when analytics.js is re-written.